### PR TITLE
clarify and expand syndic docs

### PR DESCRIPTION
**Pull request by jfindlay**

### CODE REVIEW

File: doc/topics/topology/syndic.rst [hunk 1]
```diff
@@ -4,102 +4,187 @@
...
+    find that the CLI doesn't wait long enough for the Syndics to return their
+    events.  If you think this is the case, you can set the
+    :conf_master:`syndic_wait` value in the Master configs on the Master or
+    Syndic nodes from which commands are executed.  The default value is ``5``,
+    and should work for the majority of deployments.
+
+In order for a Master or Syndic node to return information from Minions that
+are below their Syndics, the CLI requires a short wait time in order to allow
+the Syndics to gather responses from their Minions. This value is defined in
+the :conf_master:`syndic_wait` config option and has a default of five seconds.
```
Comment (basepi) [inline doc/topics/topology/syndic.rst]: This section is not quite right. The syndic isn't gathering minion responses during this time, but rather it is using the target string to generate a list of minions it expects to return for the command. (The master does this as well, but doesn't know about minions below the syndic). The `syndic_wait` is the amount of time we wait for the syndic to return this data, before assuming the syndic is dead, or is not going to return this list. If the syndic doesn't return this list of expected minions in time, then the master could assume there are no minions expected to return from below that syndic, and return on the CLI early.

Keep me posted if that doesn't make sense.

**Comment by basepi** [comment]
One note above, otherwise looks good.

**Comment by basepi** [comment]
Edited my note above for clarity.

File: doc/topics/topology/syndic.rst (hunk 1 above)
Comment (jfindlay) [inline doc/topics/topology/syndic.rst]: Sorry, I didn't actually write this part, but I'll update it. :-)

**Pull request merged**